controllers/manilashare_controller.go

```diff
@@ -384,8 +419,9 @@ func (r *ManilaShareReconciler) reconcileNormal(ctx context.Context, instance *m
 	//
 
 	// Deploy a statefulset
+	ssDef := manilashare.StatefulSet(instance, inputHash, serviceLabels, serviceAnnotations)
 	ss := statefulset.NewStatefulSet(
-		manilashare.StatefulSet(instance, inputHash, serviceLabels),
+		ssDef,
```

Comment:
same could keep the `manilashare.StatefulSet(instance, inputHash, serviceLabels, serviceAnnotations)` call here.

Reply:
+1